### PR TITLE
[azure-pipelines] update azp from buster to bullseye

### PR DESCRIPTION
**Pull request by maipbui**
Signed-off-by: maipbui <maibui@microsoft.com>

#### What I did
Update to bullseye to use python 3.9
#### How I did it
Change pipeline from buster to bullseye
#### How to verify it
Pass PR build checker

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -48,8 +48,9 @@ stages:
       displayName: "Download artifacts from latest sonic-buildimage build"
 
     - script: |
+        apt list | grep "libboost-serialization"
```
Comment (qiluo-msft) [inline azure-pipelines.yml:51]: >apt list | grep "libboost-serialization" [](http://example.com/codeflow?start=8&length=40)

This is for debug purpose? Suggest remove in final code.

File: azure-pipelines.yml [hunk 2]
```diff
@@ -48,8 +48,9 @@ stages:
...
         set -xe
-        sudo apt-get -y purge libhiredis-dev libnl-3-dev libnl-route-3-dev
+        sudo apt-get -y purge libhiredis-dev libnl-3-dev libnl-route-3-dev libboost-serialization1.71-dev
```
Comment (qiluo-msft) [inline azure-pipelines.yml:53]: >purge [](http://example.com/codeflow?start=24&length=5)

`purge` is like removing a package. Is it intended?

File: .azure-pipelines/build_and_install_module.sh [hunk 3]
```diff
@@ -29,6 +29,8 @@ function build_and_install_kmodule()
     apt-get install -y build-essential linux-headers-${KERNEL_RELEASE} autoconf pkg-config fakeroot
     apt-get install -y flex bison libssl-dev libelf-dev
     apt-get install -y libnl-route-3-200 libnl-route-3-dev libnl-cli-3-200 libnl-cli-3-dev libnl-3-dev
+    # Install libs required by libswsscommon for build
+    apt-get install -y libzmq3-dev libzmq5 libboost-serialization1.71.0 uuid-dev
```
Comment (qiluo-msft) [inline .azure-pipelines/build_and_install_module.sh:33]: >1.71.0 [](http://example.com/codeflow?start=65&length=6)

If you are build in bullseye, you want to install 1.74?

File: .azure-pipelines/build_and_install_module.sh [hunk 4]
```diff
@@ -29,6 +29,8 @@ function build_and_install_kmodule()
     apt-get install -y build-essential linux-headers-${KERNEL_RELEASE} autoconf pkg-config fakeroot
     apt-get install -y flex bison libssl-dev libelf-dev
     apt-get install -y libnl-route-3-200 libnl-route-3-dev libnl-cli-3-200 libnl-cli-3-dev libnl-3-dev
+    # Install libs required by libswsscommon for build
+    apt-get install -y libzmq3-dev libzmq5 libboost-serialization-dev uuid-dev
```
Comment (zbud-msft) [inline .azure-pipelines/build_and_install_module.sh:33]: We can remove this line now I believe

**Review by zbud-msft — APPROVED** [review]

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**